google/cloud/spanner/client.h

```diff
@@ -961,8 +961,8 @@ std::shared_ptr<spanner::Connection> MakeConnection(spanner::Database const& db,
  * The returned connection object should not be used directly, rather it should
  * be given to a `Client` instance, and methods should be invoked on `Client`.
  *
- * @note Prefer using the `MakeConnection()` overload that accepts
- *     `google::cloud::Options`.
+ * @deprecated Please use the `MakeConnection()` overload that accepts
```

Comment:
Should we add the `[[deprecated]]` attribute too?

Reply:
I planned to leave that until the old `DatabaseAdminClient` goes away.  But, maybe it would be better to try to kill them both with the same stone.  I'll think on it.